manifests/plugin/remote_execution/script.pp

```diff
@@ -49,7 +51,7 @@
 
   foreman_proxy::plugin::module { 'remote_execution_ssh':
     enabled   => $enabled,
-    feature   => 'SSH',
+    feature   => 'Script',
```

Comment:
Could it happen that someone would use newer puppet modules including this change, but have older packages where sp-rex-ssh would still provide ssh feature?

Reply:
That is possible. For example, https://github.com/theforeman/puppet-foreman_proxy/pull/739/files was just opened, and if the author is intending to use the HEAD of this module with an older Foreman they will find it breaks. In the past we've updated the README with what the module supports. I do not recall if we ever tried to do things differently depending on the Foreman version within the code.

Reply:
> I do not recall if we ever tried to do things differently depending on the Foreman version within the code.

Then as a poor man's solution let's at least expose the expected feature as a parameter so it is not a hard stop for people with incompatible mix of packages and modules?

Reply:
We will run into this same problem when we introduce the cockpit flag and the pull-mqtt support. There are enough changes that I think we may be better off releasing a latest version of the module, and bumping this to 22.Y and adjusting https://github.com/theforeman/puppet-foreman_proxy#compatibility to reflect that now it's only for 3.3 and newer. @evgeni thoughts?

Reply:
This change "only" breaks REX for older Foremans and in the past we had taken the direction of saying "this release is only *officially* supporting Foreman X+, but in reality, only featureA needs X+, others should work"

Read: I'm fine with breaking REX for pre-3.3 installs with this change.